components/indexer.py

```diff
@@ -1,56 +1,88 @@
 import rev
 import wpilib
+from enum import Enum
 from magicbot import tunable, feedback
-from typing import Tuple
 
 
 class Indexer:
-    indexer_front_motor: rev.CANSparkMax
-    indexer_back_motor: rev.CANSparkMax
+    class Direction(Enum):
+        OFF = 0
+        FORWARDS = 1
+        BACKWARDS = -1
+
+    # The "tunnel" is the horizontal part of the indexer that the cargo enters first
+    # The "chimney" is the vertical section of the indexer that feeds the shooter
+    # The "cat flap" is the moving flap that allows an opposition cargo to be captured
+    indexer_tunnel_motor: rev.CANSparkMax
+    indexer_chimney_motor: rev.CANSparkMax
     colour_sensor: rev.ColorSensorV3
-    prox_sensor1: wpilib.DigitalInput
-    prox_sensor2: wpilib.DigitalInput
-    # indexer_piston: wpilib.Solenoid
+    tunnel_prox_sensor: wpilib.DigitalInput
```

Comment:
if the second proximity sensor is serving the same purpose as before, would this be better named as lower chimney sensor?

Reply:
Yes, good point.